microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,114 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+When you select the Address bar, a signal is sent to your default search provider.  The signal tells the provider to prepare suggestions.  The typed characters and search queries aren't sent to Microsoft unless your search provider is Bing.
 
-<!-- The Privacy settings have been updated from what is described here -->
 
-If Microsoft Edge detects typing in the address bar that might contain sensitive information, it does not send the typed text. Sensitive information includes authentication credentials, local file names, or URL data that is normally encrypted.
+<!-- ------------------------------ -->
+#### Controlling whether typed characters are sent
+
+You can control whether typed characters are sent to your default search provider.  To enable sending search and site suggestions by using your typed characters in the Address bar to your default search provider:
+
+1. Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**, and then scroll down to the **Services** section.
+
+1. Click the **Address bar and search** section.  The **Address bar and search** settings page<!-- edge://settings/search --> opens.
+
+1. Click the **Search suggestions and filters** section.  The **Search suggestions and filters** settings page<!-- edge://settings/searchFilters --> opens.
+
+1. Use the toggle **Show me search and site suggestions using my typed characters**:
+
+   ![The toggle in Edge Settings for "Show me search and site suggestions using my typed characters"](./index-images/search-suggestions-typed-chars.png)
+
+   If you turn off that toggle, your typed characters are no longer sent to your default search provider, but your search queries are still sent to your default search provider, to provide search results.<!--per ui: "If you turn this off, you'll only see suggestions from your favorites and history. Typed characters won't be sent to the search engine used in the address bar." -->
+
+
+<!-- ------------------------------ -->
+#### Microsoft Search in Bing
+
+If you're signed into Microsoft Edge using your work or school account (Microsoft Entra ID account), your typed-in characters are also sent to Microsoft Search in Bing, which returns work- or school-related results and cross-device suggested queries to you.
+
+Your administrator can choose to turn on or off the work- or school-related results or query suggestions entirely for your organization, by using the `AddressBarMicrosoftSearchInBingProviderEnabled` Microsoft Search in Bing (MSB) policy.  If your administrator turns off the `AddressBarMicrosoftSearchInBingProviderEnabled` policy:
```

Comment:
Admin policies (and their effects) are irrelevant to the public privacy whitepaper
```suggestion
```

Reply:
I'm Resolving the 7 @apryzgoda comments about the policy passage without deleting the passage, because the comments are Outdated (the surrounding markdown is an old commit) and b/c Dexter wants to keep the passage.

Alyssa, if you want, you can enter a fresh Comment in a fresh Files Changed tab w/ short url: 
https://github.com/MicrosoftDocs/edge-developer/pull/3224/files
To delete a group of lines as a single Comment, you can click to the right of a line number and drag down (to get a yellow vertical bar), then empty-out the triple-tickmarks.